department-of-veterans-affairs/caseflow
# MTV | Backend Support for "Return to Litigation Support" Workflow

Connects #12525

### Description
Added backend support for the "Return to Litigation Support" workflow whereby a judge cancels `AddressMotionToVacate` task and reopens `VacateMotionMailTask` with instructions to include missing draft.

### Acceptance Criteria

### Testing Plan
1. As a judge, go to an assigned `JudgeAddressMotionToVacateTask`
2. Click "return to Litigation Support team" link
3. Complete modal
4. Ensure that tasks have been updated
